go/teams/loader.go

```diff
@@ -1729,18 +1740,13 @@ func (l *TeamLoader) logIfUnsyncedSecrets(ctx context.Context, state *keybase1.T
 	}
 }
 
-func (l *TeamLoader) lows(mctx libkb.MetaContext, state *keybase1.TeamData, hp *hidden.LoaderPackage) getLinksLows {
+func (l *TeamLoader) lows(mctx libkb.MetaContext, state *keybase1.TeamData, hp *hidden.LoaderPackage, arg load2ArgT) getLinksLows {
 	var lows getLinksLows
 	if state != nil {
 		chain := TeamSigChainState{inner: state.Chain}
 		lows.Seqno = chain.GetLatestSeqno()
-		lows.PerTeamKey = keybase1.PerTeamKeyGeneration(len(state.PerTeamKeySeedsUnverified))
-		// Use an arbitrary application to get the number of known RKMs.
-		// TODO: using an arbitrary RKM is wrong and could lead to stuck caches.
-		//       See CORE-8445
-		rkms, ok := state.ReaderKeyMasks[keybase1.TeamApplication_CHAT]
-		if ok {
-			lows.ReaderKeyMask = keybase1.PerTeamKeyGeneration(len(rkms))
+		if !arg.foundRKMHole {
+			lows.PerTeamKey = keybase1.PerTeamKeyGeneration(len(state.PerTeamKeySeedsUnverified))
```

Comment:
It could be that the reason both `len(PerTeamKeySeedsUnverified)` and `len(rkms)` were used before was in case the cache held a team with no chat RKMs but did have PTKs. Such as would be the case for an implicit admin who then is upgraded to an explicit admin. This is pure speculation. Make sure there's a test or some reasoning that covers that.

Oh but since neither value was actually used it matters much less why they were how they were.

Reply:
Can I talk you out of tests?